models/ideas.js

```diff
@@ -14,6 +15,19 @@ function createIdea(header, description, askedSum, userId) {
     return newUser;
 }
 
+function updateIdea(ideaId, idea) {
+    let ideas = getIdeas();
+    const idea = idea;
```

Comment:
Why do we need this?

Reply:
We don't, trying to wright down it another way